### ISSUE TITLE
Add page title and <h1> to Session timeout page

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#8539]
## Area or Component

[Session timeout](https://github.com/CDCgov/prime-simplereport/blob/4f2d9ee0ed0b279e23ab5086a0cea1f616719d18/frontend/src/app/accountCreation/SessionTimeout.tsx)

## Quick Summary

Text appears and functions like a section heading but is not marked up as such.

## Accessibility Violation

https://docs.deque.com/issue-help/1.0.0/en/semantic-heading

## Action Requested

- Add a `<h1>` heading
- Add a page title ([for example, like this page](https://github.com/CDCgov/prime-simplereport/blob/4f2d9ee0ed0b279e23ab5086a0cea1f616719d18/frontend/src/app/accountCreation/PasswordForm/PasswordForm.tsx#L26))

## Source Code (optional)

[which file or line is the issue attached to?]: #
